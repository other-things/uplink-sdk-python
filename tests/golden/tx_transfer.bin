00000000: 00 01 00 01 2D 37 7F D1  0E 64 34 AF 54 AC E3 A3  ....-7...d4.T...
00000010: 8E 14 52 C3 1A 09 43 0E  82 50 58 09 A9 CA 5D 69  ..R...C..PX...]i
00000020: EF 24 94 C6 64 86 E7 77  43 09 64 01 FD AF C6 D7  .$..d..wC.d.....
00000030: 60 D8 90 4E 01 39 79 5F  74 00 84 A8 51 2F 92 75  `..N.9y_t...Q/.u
00000040: 48 66 82 1D 00 00 00 00  00 00 00 00 00 05        Hf............