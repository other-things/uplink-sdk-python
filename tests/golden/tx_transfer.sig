r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 68085176034252722895944718105491693363200230653664532663664457739254383236920
