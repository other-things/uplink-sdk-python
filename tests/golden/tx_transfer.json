{
    "header": {
        "contents": {
            "contents": {
                "assetAddr": "43WRxMNcnYgZFcE36iohqrXKQdajUdAxeSn9mzE1ZedB",
                "balance": {
                    "decimalIntegerValue": 5,
                    "decimalPlaces": 0,
                    "tag": "Dec"
                },
                "toAddr": "7mR5d7s6cKB4qjuX1kiwwNtygfURhFQ9TKvEd9kmq6QL"
            },
            "tag": "CreateAsset"
        },
        "tag": "TxAsset"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE02ODA4NTE3NjAzNDI1MjcyMjg5NTk0NDcxODEwNTQ5MTY5MzM2MzIwMDIzMDY1MzY2NDUzMjY2MzY2NDQ1NzczOTI1NDM4MzIzNjkyMA=="
}